optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -90,8 +90,11 @@ def allocate_kv_cache(self, batch_size, seq_len):
         if self.past_key is None or self.past_key.shape != key_shape:
             device = self.k_proj.weight.device
             dtype = self.k_proj.weight.dtype
-            self.past_key = torch.empty(key_shape, dtype=dtype, device=device)
-            self.past_value = torch.empty(value_shape, dtype=dtype, device=device)
+            self.past_key = torch.zeros(key_shape, dtype=dtype, device=device)
+            self.past_value = torch.zeros(value_shape, dtype=dtype, device=device)
```

Comment:
@puneeshkhanna  `torch.zeros` lead to more memory consumed when running with FP8 in 1.13.0, can you help to double check?